### DISCUSSION TITLE
UPDATE MOBILE & DESKTOP UNIVERSAL LINK FOR ANCRYPTO WALLET APP SUBMISSION #360

### DISCUSSION THREAD

**Discussion by antiergit** [WalletConnect/walletconnect-monorepo#946] [Developer Support]
Hi @chadyj, I want to update the universal link to our wallet connect registry application. Kindly advise on how I can update it, as the issue has already been closed and archived by you. Thanks.

Issue Link: https://github.com/WalletConnect/walletconnect-registry/issues/360

I have already created an account on Discord and wallet connect with email apple@antech.in, unfortunately I was unable to find you on Discord.

Thank You

**Comment by chadyj** [comment]
I'll help you get this sorted.
If you can't find me in discord, just post #support and I'll follow up with you there.
